cli.rb

```diff
@@ -1,33 +1,26 @@
 # frozen_string_literal: true
 require 'thor'
 require 'exif'
+require_relative 'writers/csv_report_writer'
+require_relative 'writers/xml_report_writer.rb'
+require_relative 'exif_scanner.rb'
 
 class CLI < Thor
   desc "scan", "Recursively reads all of the images from the supplied directory"
   option :path, type: :string, required: false, desc: "Optional parameter that allows any other directory to be passed in."
+  option :format, required: false, default: :csv, desc: "Optional parameter that allows setting specific file formats, either CSV or XML. The default is CSV."
 
   def scan
-    File.open("report.csv", "w") do |report|
-      Dir.glob("**/*").each do |file|
-        next unless File.file?(file)
+    report = ExifScanner::gps(options[:path])
```

Comment:
Awesome, I love how you extracted this into separate files! 

When calling class/static methods, while you can use the `::` syntax, it's more common to just use `.` 

```suggestion
    report = ExifScanner.gps(options[:path])
```